### ISSUE TITLE
Route for project/${project.id} is dead

### ISSUE THREAD

**Issue by hcwiley** [prescottprue/generator-react-firebase#82]
**Do you want to request a *feature* or report a *bug*?**  
bug

**What is the current behavior?**
Project page has no data from firebase store project.

**What is the expected behavior?**
It should display the project name and then the json below that

**Which versions of dependencies, and which browser and OS are affected by this issue? Did this work in previous versions or setups?**
I would image all browsers. I don't know about previous versions.

**Comment by hcwiley** [comment]
I'm not sure why this is using [firebaseConnect](https://github.com/prescottprue/generator-react-firebase/blob/master/examples/redux-firestore/src/routes/Projects/routes/Project/components/ProjectPage/ProjectPage.enhnacer.js#L6) when the project is made using [firestore](https://github.com/prescottprue/generator-react-firebase/blob/master/examples/redux-firestore/src/routes/Projects/components/ProjectsPage/ProjectsPage.enhancer.js#L56)

**Comment by hcwiley** [comment]
to be clear, i ran ```yo react-firebase``` and did the additional setup with material, firestore.

**Comment by prescottprue** [comment]
@hcwiley You are correct, when selecting Firestore, it should use `firestoreConnect` instead of `firebaseConnect.

**Comment by hcwiley** [comment]
I've changed the [ProjectPage.enhancer.js](https://github.com/prescottprue/generator-react-firebase/blob/master/examples/redux-firestore/src/routes/Projects/routes/Project/components/ProjectPage/ProjectPage.enhnacer.js) to look like this and it is working:

```js
import { compose } from 'redux'
import { connect } from 'react-redux'
import { firestoreConnect, getVal } from 'react-redux-firebase'
import { spinnerWhileLoading } from 'utils/components'
import { UserIsAuthenticated } from 'utils/router'

export default compose(
  // redirect to /login if user is not logged in
  UserIsAuthenticated,
  // Map auth uid from state to props
  connect(({ firebase: { auth: { uid } } }) => ({ uid })),
  // Wait for uid to exist before going further
  spinnerWhileLoading(['uid']),
  // Create listeners based on current users UID
  firestoreConnect(({ params, uid }) => [
    // Listener for projects the current user created
    {
      collection: 'projects',
      where: ['createdBy', '==', uid]
    }
  ]),
  spinnerWhileLoading(['firestore']),
  // // Map projects from state to props
  connect(({ firestore: { data } }, {params}) => ({
    project: getVal(data, `projects/${params.projectname}`)
  })),
  // Show loading spinner while project is loading
  spinnerWhileLoading(['project'])
)
```

**Comment by prescottprue** [comment]
@hcwiley The changes that you mentioned have been added, and published [in `v2.0.0-rc.2`](https://github.com/prescottprue/generator-react-firebase/releases/tag/v2.0.0-rc.2). Don't forget to update using `npm i -g generator-react-firebase` otherwise you will not get the changes.

Let me know if it doesn't work as intended for you, thanks for reporting!
